geom/twkb.go

```diff
@@ -0,0 +1,50 @@
+package geom
+
+// Tiny Well Known Binary
+// See spec https://github.com/TWKB/Specification/blob/master/twkb.md
+
+const (
+	twkbTypePoint              = 1
```

Comment:
nit: These constants could technically be successfully interchanged with one another. That is, usages of `twkbTypeMultiPoint ` could be accidentally but successfully replaced with `twkbMaxDimensions` or `twkbHasIDs` and succeed. Maybe we can consider typing them?

```
type geomKinds int
type metaHeaders int
```

Reply:
I've introduced types for the geometry kinds and metadata headers. You're right, I think it's cleaner and more comprehensible this way.